NewForce-at-Mountwest/bangazonapi-teamoddoose
# Product Type Controller creation - Ticket #5

## Proposed changes

**ProductType Controller** functions:

GET ALL entities in the ProductType table in Database, 
GET a SINGLE entity in the ProductType table, 
POST an entity into the ProductType table, 
PUT / Edit an existing entity in the ProductType table
DELETE a single entity in the ProductType table. 

## Related Issues

## Steps To Test

GitBash: CD into BangazonAPI directory and type in the command 'dotnet run'. Once the program is running, open up the Postman desktop app and run the following commands:

Select GET then paste http://localhost:63540/api/ProductTypes/ into the field and click send. The result should list all ProductTypes in the database.

Select GET then paste http://localhost:63540/api/ProductTypes/1 or any other number that showed up in the previous query as ProductTypeId and click send. The result should be only that object of the specified ProductType

Select POST, then paste http://localhost:63540/api/ProductTypes/ into the field, then click Body underneath the field, then select raw, and then paste this snippet or make one similar
{
"name":"cheese"
}
then click send. The result will be the new ProductType.

Select PUT then paste http://localhost:63540/api/ProductTypes/1 or any other ProductTypeId then do the same as the POST example and change the values, click send. Nothing will populate. When you run the GET query the computer you specified in your PUT query should show the updated, edited information you gave it.

Select DELETE then http://localhost:63540/api/ProductTypes/2 or any other ProductTypeId then click send. When the GET query is submitted, the computer you specified in your DELETE query should no longer exist.

Checklist:


## What kind of change does this PR introduce?
Please check one of the following:
- [x ] New feature
